clamonacc/inotif/inotif.c

```diff
@@ -236,13 +245,15 @@ static int onas_ddd_watch_hierarchy(const char *pathname, size_t len, int fd, ui
         size_t size      = len + strlen(curr->dirname) + 2;
         char *child_path = (char *)cli_malloc(size);
         if (child_path == NULL)
+            logg(LOGG_ERROR, "ClamInotif: out of memory when when adding child for %s\n", hnode->pathname);
```

Comment:
You're missing `{ }` curlies here, so it fails to watch any directories with CL_EMEM.